src/Scientist/Scientist.csproj

```diff
@@ -10,8 +10,8 @@
     <Authors>GitHub</Authors>
     <Description>Implementation of the Scientist library from GitHub</Description>
```

Comment:
```suggestion
    <Description>Implementation of the Scientist library for .net, inspired by the Ruby library</Description>
```

Reply:
You are going to bring The Wrath of Immo upon us with that spelling 😁 	

How does
`Implementation of the Scientist library for .NET, inspired by the Ruby library from GitHub`
sound?

Reply:
> You are going to bring The Wrath of Immo upon us with that spelling 😁

Paging @terrajobst to this very urgent bike shed discussion :trollface:

Reply:
> Implementation of the Scientist library for .NET, inspired by the Ruby library from GitHub

:shipit: